src/content/en/updates/2016/10/navigator.share.md

```diff
@@ -17,10 +17,10 @@ platform.
 There have been a number of ways to invoke native sharing capabilities on the
 platform, but they all have significant drawbacks.  There was [Web
 Intents](https://en.wikipedia.org/wiki/Paul_Kinlan){: .external } (dead), there
-is protocol handling (poor support on mobile), there is direct sharing to a
-well-known service URL such as Twitter's, and there is also the [Android intent:
-URL syntax](https://paul.kinlan.me/sharing-natively-on-android-from-the-web/)
-{: .external } (which was, unfortunately, Android-only, and required apps to opt-
+is protocol handling via `registerProtocolHandler` but this has zero support on
+mobile, there is direct sharing to a well-known service URL such as Twitter's,
+and there is also the [Android intent: URL syntax](https://paul.kinlan.me/sharing-natively-on-android-from-the-web/) {:
```

Comment:
No space between the end of the markdown url and the `{: .external }`, otherwise it doesn't apply the class.